iam-persistence/src/main/resources/db/migration/test/V100000_6__add_aarc_test_client_and_scopes.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO system_scope(scope, description, icon, restricted, default_scope, structured, structured_param_description) VALUES
+  ('eduperson_scoped_affiliation', 'aarc organisation name', 'null', false, false, false, null),
+  ('eduperson_entitlement', 'aarc groups', 'null', false, false, false, null);
```

Comment:
Change descriptions to:

- eduperson_scoped_affiliation:  "Access to EduPerson scoped affiliation information"
- eduperson_entitlement: "Access to EduPerson entitlements information"

Reply:
Done. And also moved this INSERT into both h2 and mysql migrations.